what4/src/What4/Solver/Yices.hs

```diff
@@ -436,7 +436,7 @@ newConnection stream ack reqFeatures timeout bindings = do
   let nlSolver = reqFeatures `hasProblemFeature` useNonlinearArithmetic
   let features | efSolver  = useLinearArithmetic
                | nlSolver  = useNonlinearArithmetic .|. useIntegerArithmetic
-               | otherwise = yicesDefaultFeatures
+               | otherwise = reqFeatures
```

Comment:
This looks like the right fix, but we should probably also add `useIntegerArithmetic` to the default features.

Reply:
This snuck in by accident, this is orthogonal to the bound variable issue.